src/renderer/core/forwarder/forwarderMailer.ts

```diff
@@ -1,3 +1,5 @@
+/* eslint-disable @typescript-eslint/camelcase */
```

Comment:
Too broad, see explanation above.

Reply:
In this case I suggest only wrapping the affected code with [block comments](https://eslint.org/docs/2.13.1/user-guide/configuring#disabling-rules-with-inline-comments). Your first change here disabled the rule for the whole file (which was too broad). 
```
/* eslint-disable @typescript-eslint/camelcase */
const payload: ForwarderMessage = {
  ...
};
/* eslint-enable @typescript-eslint/camelcase */
```